content/influxdb/cloud-iox/select.md

```diff
@@ -0,0 +1,114 @@
+---
+title: SELECT statement
+description: >
+  Use the SQL `SELECT` statement to query data from a measurement.
+menu:
+  influxdb_cloud_iox:
+    name: SELECT statement
+    parent: Explore data using SQL
+weight: 210
+---
+
+Use the `SELECT` statement to query data from an InfluxDB measurement.
+The `SELECT` clause is required when querying data in SQL.
+
+- [Syntax](#syntax)
+- [Examples](#examples)
+
+### Syntax
+
+Basic syntax:
+
+```sql
+SELECT a, b, "time" FROM <measurement>
+```
+
+{{% note %}}
+**Note:** The `SELECT` statement **always requires** a `FROM` clause.
+{{% /note %}}
+
+The SELECT clause supports the following:
+
+ - `SELECT *` - return all tags, fields and timestamps.
+ - `SELECT DISTINCT` to return all distinct (different) values. 
+ - `SELECT <"field" or "tag">` - returns a specified field or tag.
+ - `SELECT <"field" or "tag">, <"field" or "tag">` - returns more than one tag or field.
+ - `SELECT <"field"> AS a `- return the field as the alias.
+
+## Examples
+
+The following examples use data from the NOAA database. 
+To download the NOAA test data see [NOAA water sample data](/influxdb/v2.6/reference/sample-data/#noaa-water-sample-data).
+
+### Select all fields and tags from a measurement
+
+```sql
+SELECT * FROM h2o_feet
```

Comment:
it might be useful to include `limit 10` to help people avoid selecting some massive result when they copy/paste/modify

```suggestion
SELECT * FROM h2o_feet LIMIT 10
```